### PR TITLE
[#767] ignored JavaRefactoringIntegrationTest on Oxygen

### PR DESCRIPTION
**Pull request by cdietrich**
[#767] ignored JavaRefactoringIntegrationTest on Oxygen

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtend.ide.tests/longrunning/src/org/eclipse/xtend/ide/tests/refactoring/JavaRefactoringIntegrationTest.java [hunk 1]
```diff
@@ -12,20 +12,31 @@
...
 /**
  * @author Jan Koehnlein - Initial contribution and API
  */
 public class JavaRefactoringIntegrationTest extends AbstractXtendRenameRefactoringTest {
 
+	@Before
+	public void doNotRunOnOxygen() {
+		//FIXME workaround for https://github.com/eclipse/xtext-xtend/issues/767
+		Version version = Platform.getBundle("org.eclipse.core.runtime").getVersion();
+		Assume.assumeFalse(version.getMajor() == 3 && version.getMinor() == 13);
```
Comment (kthoms) [inline org.eclipse.xtend.ide.tests/longrunning/src/org/eclipse/xtend/ide/tests/refactoring/JavaRefactoringIntegrationTest.java]: Should this be checked for one platform version or can minor be checked for lower/greater equals?

Comment (cdietrich) [reply org.eclipse.xtend.ide.tests/longrunning/src/org/eclipse/xtend/ide/tests/refactoring/JavaRefactoringIntegrationTest.java]: i assume the likelyhood of not failing is better on > oxygen thus i test for oxgen only

Comment (kthoms) [reply org.eclipse.xtend.ide.tests/longrunning/src/org/eclipse/xtend/ide/tests/refactoring/JavaRefactoringIntegrationTest.java]: maybe add a comment that this is testing for Eclipse 4.7/Oxygen. Who should know from reading this here that 3.13 is Oxygen?

Comment (cdietrich) [reply org.eclipse.xtend.ide.tests/longrunning/src/org/eclipse/xtend/ide/tests/refactoring/JavaRefactoringIntegrationTest.java]: see commit message

Comment (kthoms) [reply org.eclipse.xtend.ide.tests/longrunning/src/org/eclipse/xtend/ide/tests/refactoring/JavaRefactoringIntegrationTest.java]: yes, and also the method's name. I meant a code comment that the magic number 3.13 _is_ actually Oxygen.

**Pull request merged**